### PR TITLE
Update Online/Offline Installer for Observability

### PR DESCRIPTION
**Pull request by taohe1012**
# Description
Documentation update of Installer and offline installer for CSM
Observability deployment:
 - online installer support for Observability PowerScale;
 - online installer support for Observability PowerScale Authorization;
 - update offline installer documentation for Observability PowerScale;
 - update offline installer for Observability to enable Authorization;
 - add upgrade steps for offline installer in documentation.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/422|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [x] I have made corresponding changes to the documentation
- [x] I have maintained backward compatibility

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [x] Test A: manual test for online installer: the Observability including powerscale is installed and upgraded successfully.

- [x] Test B: manual test for online installer: the Observability with enabling powerscale/powerflex Authorization is installed and upgraded successfully.

- [x] Test C: manual test for offline installer: the Observability with powerscale/powerflex Authorization is installed and upgraded successfully.

### CODE REVIEW

File: installer/README.md [hunk 1]
```diff
@@ -266,30 +273,106 @@ Example command to copy the Secret from the csi-powerstore namespace to the kara
...
+
+You need to provide your own [configurations](../docs/GETTING_STARTED_GUIDE.md#Configuration). A sample values.yaml file is located [here](https://github.com/dell/helm-charts/blob/main/charts/karavi-observability/values.yaml).
+
+In your own configuration values.yaml, you need to enable PowerFlex/PowerScale Authorization, and provide the location of the sidecar-proxy Docker image and URL of the Karavi Authorization proxyHost address.
+
+4. Now that the required images have been made available and the Helm chart's configuration updated with references to the internal registry location, installation can proceed by following the instructions that are documented within the Helm chart's repository.
+
+**Note**: Assuming that Your Secrets from CSI Drivers have been copied to the Karavi Observability namespace in the phase of `#Offline Karavi Observability Helm Chart Installer`.
+
+**Note**: Optionally, you could provide your own [configurations](../docs/GETTING_STARTED_GUIDE.md#Configuration). A sample values.yaml file is located [here](https://github.com/dell/helm-charts/blob/main/charts/karavi-observability/values.yaml).
```
Comment (baoy1) [inline installer/README.md:366]: Is `../docs/GETTING_STARTED_GUIDE.md#Configuration` valid location?

Comment (taohe1012) [reply installer/README.md]: @baoy1  updated, its link is removed

**Review by YianZong — APPROVED** [review]

**Review by forrestxia — DISMISSED** [review]

**Review by baoy1 — APPROVED** [review]

**Review by P-Cao — DISMISSED** [review]

File: installer/README.md [hunk 2]
```diff
@@ -266,30 +273,106 @@ Example command to copy the Secret from the csi-powerstore namespace to the kara
...
+## Workflow
+
+To perform an offline installation of a helm chart, the following steps should be performed:
+
+1. Build an offline bundle.
+2. Unpack the offline bundle and prepare for installation.
+3. Perform a Helm upgrade.
+
+### Build the Offline Bundle
+Follow `# Offline Karavi Observability Helm Chart Installer`, Build the latest bundle.
```
Comment (shaynafinocchiaro) [inline installer/README.md:337]: Can this link to the` Build the Offline Bundle` steps?

File: installer/README.md [hunk 3]
```diff
@@ -266,30 +273,106 @@ Example command to copy the Secret from the csi-powerstore namespace to the kara
...
+
+### Unpack the Offline Bundle
+Follow `# Offline Karavi Observability Helm Chart Installer`, Copy and Unpack the Offline Bundle to another Linux system, and Push Docker images to the internal Docker registry.
```
Comment (shaynafinocchiaro) [inline installer/README.md:340]: Can this link to the `Unpack the Offline Bundle` steps?

File: installer/README.md [hunk 4]
```diff
@@ -266,30 +273,106 @@ Example command to copy the Secret from the csi-powerstore namespace to the kara
...
+2. Install necessary cert-manager CustomResourceDefinitions provided.
 ```
-[user@anothersystem /home/user/offline-karavi-observability-bundle/helm]# kubectl get secrets,deployments -n karavi -o yaml | karavictl inject --insecure=false --root-certificate <location-of-root-certificate> --image-addr <sidecar-proxy-image-location> --proxy-host <proxy-host> | kubectl apply -f -
+[user@anothersystem /home/user/offline-karavi-observability-bundle/helm]# kubectl apply --validate=false -f cert-manager.crds.yaml
+```
+
+3. (Optional) The following steps can be performed to enable Karavi Observability for PowerFlex/PowerScale to use an existing instance of Karavi Authorization for accessing the REST API for the given storage systems.
+
+**Note**: Assuming that If the Karavi Observability's Authorization has beed enabled in the phase of `#Offline Karavi Observability Helm Chart Installer`, the Authorization Secrets/Configmap have been copied to the Karavi Observability namespace.
```
Comment (shaynafinocchiaro) [inline installer/README.md:356]: Misspelt been in `has beed enabled`

File: installer/README.md (hunk 2 above)
Comment (taohe1012) [reply installer/README.md]: @shaynafinocchiaro  thanks, its link is added

File: installer/README.md (hunk 3 above)
Comment (taohe1012) [reply installer/README.md]: @shaynafinocchiaro  thanks, its link is added

File: installer/README.md (hunk 4 above)
Comment (taohe1012) [reply installer/README.md]: @shaynafinocchiaro  thanks, updated

**Review by shaynafinocchiaro — APPROVED** [review]

**Review by sharmilarama — APPROVED** [review]

**Review by P-Cao — APPROVED** [review]

**Review by baoy1 — APPROVED** [review]

**Review by forrestxia — APPROVED** [review]

**Review by YianZong — APPROVED** [review]

**Pull request merged**